apps/alpha-test-app/src/components/layout-demo/SingleSettingsGroup.vue

```diff
@@ -0,0 +1,44 @@
+<script lang="ts" setup>
+import { OnyxRadioButtonGroup, type SelectionOption } from "sit-onyx";
+import { computed } from "vue";
+import type { Settings } from "./LayoutSettings.vue";
+
+const activeSetting = defineModel<Settings>();
+
+const props = defineProps<{
+  headline: string;
+  options: SelectionOption<undefined>[];
+  horizontal?: boolean;
+}>();
+
+const options = computed<SelectionOption<Settings>[]>(() =>
+  props.options.map(
+    (option) => ({ ...option, value: { [option.id]: true } }) as SelectionOption<Settings>,
+  ),
+);
+
+const settingsToSelection = (settings?: Settings): SelectionOption<Settings> | undefined => {
+  return settings
+    ? options.value.find((option) => option.id === Object.keys(settings)[0])
```

Comment:
This doesn't work if the value for the first key is false, right? Shouldn't only true values be included?

Reply:
done